src/components/views/auth/ServerConfig.js

```diff
@@ -244,7 +266,7 @@ export default class ServerConfig extends React.PureComponent {
         });
         return <div className={classes}>
             {_t("Enter your custom identity server URL <a>What does this mean?</a>", {}, {
-                a: sub => <a className="mx_ServerConfig_help" href="#" onClick={this.showHelpPopup}>
+                a: sub => <a className="mx_ServerConfig_help" href="#" onClick={this.showIsHelpPopup}>
```

Comment:
please let's stop using `is` as a shorthand for `identity server` - `identity` is just as fine, and doesn't come across as booleans.